markdown/2017-10-02-welcome.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Weekly Digest
+show_social: true
+show_agenda: true
+show_sponsorship: true
+...
+
+---
+
+Hello, and welcome to another year at the Department of Computing!
+<br>Regardless of whether you're a returning student or just started, this
+(albeit rather sizeable) email contains incredibly useful information, so keep
+on reading!
+
+---
+
+# What is DoCSoc?
+DoCSoc is the Department of Computing Society at Imperial College London!
+
+This year it is brought to you by:
+
+- **President**: Maurice Yap
+- **Vice President**: Thomas Bower
+- **Treasurer**: Levente Kurusa
+- **Secretary**: Nik Vangerow (that's me!)
+- **Academic Events Coordinator**: Rachel Mekhtieva Lee
+- **Industrial Events Coordinator**: Yoon Kim
+- **Social Events Coordinator**: Diana Avram
+- **Events Officers**: Alessandro Bonardi, Jack Pordi, Rohan Pritchard
+- **Marketing Officer**: Inara Ramji
+- **Webmaster**: Elliot Greenwood
+- **Game Dev Officer**: Uno Perwironegoro
+
+Throughout the year you will also get to know your departmental representatives
+(Dep Reps). They represent the student body (you) to the department.
+
+This year they are:
+
+- **Computing**: Martin Zlocha
+- **Joint Maths and Computing (JMC)**: Rachel Mekhtieva Lee
+- **Electronics and Information Engineering (EIE)**: Mwanakombo Hussein
+
```

Comment:
Add email addresses (either personal or the generic depreps address)

Reply:
Also did we tell the EIE deprep we're including him here ?

Reply:
Good idea 
And no, I didn't do that. Should probably do that